### ISSUE TITLE
Fail to get signers from block signer SC.

### ISSUE THREAD

**Issue by ngtuna** [BuildOnViction/tomochain-v1#356]
```ERROR[12-20|11:07:13] Fail to get signers from block signer SC. error="abi: unmarshalling empty output"```

**Comment by thanhson1085** [comment]
Etienne [6:39 PM]
```
masternode_1_91f3c7f7e018 | INFO [12-28|11:27:49] Imported new chain segment               blocks=1 txs=154 mgas=11.994 elapsed=25.470s mgasps=0.471 number=498593 hash=a62f9e…86fc29 cache=94.13mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:28:07] Imported new chain segment               blocks=1 txs=95  mgas=7.410  elapsed=18.492s mgasps=0.401 number=498594 hash=84f9aa…f14cfc cache=94.38mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:28:26] Imported new chain segment               blocks=1 txs=108 mgas=8.420  elapsed=18.771s mgasps=0.449 number=498595 hash=bddf47…2c9408 cache=94.70mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:28:26] It's time to update new set of masternodes for the next epoch... 
masternode_1_91f3c7f7e018 | ERROR[12-28|11:28:31] Error when update masternodes set. Stopping node err="abi: unmarshalling empty output"
```
when this happen, the container doesn't shutdown
getting synced is a real pain in the ass
and I see everybody in the masternode support group really losing their patience

Etienne [6:41 PM]
after restarting, it start syncing again from the start

```masternode_1_91f3c7f7e018 | INFO [12-28|11:27:49] Imported new chain segment               blocks=1 txs=154 mgas=11.994 elapsed=25.470s mgasps=0.471 number=498593 hash=a62f9e…86fc29 cache=94.13mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:28:07] Imported new chain segment               blocks=1 txs=95  mgas=7.410  elapsed=18.492s mgasps=0.401 number=498594 hash=84f9aa…f14cfc cache=94.38mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:28:26] Imported new chain segment               blocks=1 txs=108 mgas=8.420  elapsed=18.771s mgasps=0.449 number=498595 hash=bddf47…2c9408 cache=94.70mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:28:26] It's time to update new set of masternodes for the next epoch... 
masternode_1_91f3c7f7e018 | ERROR[12-28|11:28:31] Error when update masternodes set. Stopping node err="abi: unmarshalling empty output"

# docker-compose stop
Stopping node_masternode_1_91f3c7f7e018 ... done
Stopping node_metrics_1_dfbf6427dceb    ... done
# docker-compose start
Starting masternode ... done
Starting metrics    ... done
# docker-compose logs -f --tail 10 masternode

Attaching to node_masternode_1_91f3c7f7e018
[...]
masternode_1_91f3c7f7e018 | INFO [12-28|11:39:30] Block synchronisation started 
masternode_1_91f3c7f7e018 | INFO [12-28|11:39:43] Imported new chain segment               blocks=1 txs=161 mgas=12.537 elapsed=12.244s mgasps=1.024 number=498448 hash=080ea0…9306da cache=890.57kB
masternode_1_91f3c7f7e018 | INFO [12-28|11:39:57] Imported new chain segment               blocks=1 txs=75  mgas=5.856  elapsed=14.354s mgasps=0.408 number=498449 hash=2f614f…c5239d cache=1.35mB
masternode_1_91f3c7f7e018 | INFO [12-28|11:40:08] Imported new chain segment               blocks=1 txs=115 mgas=8.964  elapsed=11.094s mgasps=0.808 number=498450 hash=fb59ec…b27bf1 cache=2.03mB
```

**Comment by ngtuna** [comment]
solved via #400 #397
